README.md

```diff
@@ -1,5 +1,4 @@
-OMA Lightweight M2M IoT Agent: Overview
-==================
+# FIWARE IoT Agent for OMA LightWeight Machine2Machine
```

Comment:
Assuming that PR https://github.com/telefonicaid/lightweightm2m-iotagent/pull/147 is going to be merged before this one, a link to docker/README.md should be added.

Reply:
Fixed - link added - 7f8a2d7c